docs/src/examples/allofus.md

```diff
@@ -0,0 +1,64 @@
+
+Each Workspace will be assigned a bucket for storage on Google Cloud, that can be found on the right hand panel of the `About` page of your workspace. Each Workspace will also contain Google Cloud-specific credentials in order to submit jobs via the Google Lifesciences API. These can be found in your Workspace-specific nextflow profile, located at `~/.nextflow/config`, and available by using the flag `-profile gls` when you run nextflow. TarGene requires some additional configuration to run on the AoU Researcher workbench that is built into the `allofus` profile, which can be combined with your Workspace-specific `gls` configuration to batch out jobs when running TarGene on this platform.
+
+We reccommend running this by first entering a `Cloud Analysis Terminal` on your current Workspace, creating a configuration for the analysis you would like to run, and running TarGene in a screen session. See [`Workflows in the All of Us Researched Workbench`](https://support.researchallofus.org/hc/en-us/articles/4811899197076-Workflows-in-the-All-of-Us-Researcher-Workbench-Nextflow-and-Cromwell) for more information.
+
+A minimalist run configuration to run a flat config run on the AoU Researcher Workbench using TarGene looks like the following:
+
+```conf
+params {
+    COHORT = "ALLOFUS"
+    ESTIMANDS_CONFIG = "allofus_config.yaml"
+
+    // All of Us srWGS data
+    BGEN_FILE = "gs://fc-aou-datasets-controlled/v7/wgs/short_read/snpindel/clinvar_v7.1/bgen/clinvar.chr{1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22}.{bgen,sample,bgen.bgi}"
+    BED_FILES = "gs://fc-aou-datasets-controlled/v7/wgs/short_read/snpindel/clinvar_v7.1/plink_bed/clinvar.chr{1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22}.{bed,bim,fam}"
+    TRAITS_DATASET = "allofus_traits.csv"
+}
+```
+
+Apart from the data related parameters, there are two main parameters here: `ESTIMANDS_CONFIG` and the `ESTIMATORS_CONFIG`. These parameters describe the estimands (questions of interest) and how to estimate them respectively. 
+
+The `ESTIMANDS_CONFIG` here follows the same format as the `flat` configuration detailed in the `PheWas` section. Here we are estimating the ATE of the FTO variant (encoded as chr16:53767042:T:C in the BGEN data for the AoU cohort) on the traits present in our `allofus_traits.csv` file. We have also added `sex_at_birth` as a covariate. 
+
+```yaml
+type: flat
+
+estimands:
+  - type: ATE
+
+variants:
+  - chr16:53767042:T:C
+
+outcome_extra_covariates:
+  - "sex_at_birth"
+```
+
+The optional `outcome_extra_covariates` are variables to be used as extra predictors of the outcome (but not as confounders). This information must be contained in the `allofus_traits.csv` file, along with your outcomes-of-interest. 
+
+The `allofus_traits.csv` might look as follows:
+
+| SAMPLE_ID | sex_at_birth | Height (cm) |
```

Comment:
The sex_at_birth in the table does not seem to render, maybe with backslash? sex\_at\_birth